scenes/test_string.txt

```diff
@@ -62,6 +62,20 @@
 *gosub test_finish result_expected cslib_ret
 
 
+*comment :::::: INDEX tests ::::::
+*gosub test_start "INDEX #1"
+*temp result_expected 3
+*gosub_scene cslib_string index "ABCDEF" "C"
+*gosub test_finish result_expected cslib_ret
+
+*gosub test_start "INDEX #2"
```

Comment:
I just created this simple example of catching *bug. Feel free to use it for more serious stuff!